service/src/main/java/de/kreggel/auth/service/registration/RegistrationResource.java

```diff
@@ -0,0 +1,34 @@
+package de.kreggel.auth.service.registration;
+
+import de.kreggel.auth.core.api.UserService;
+import de.kreggel.auth.core.persistence.jpa.entity.User;
+import de.kreggel.auth.service.authentication.Credentials;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+/**
+ * Created by oliverklette on 1/2/16.
+ */
+
+    @Path("/registration")
+    public class RegistrationResource {
+        @POST
+        @Produces("application/json")
+        @Consumes("application/json")
+        public Response authenticateUser(Userdata userdata) {
```

Comment:
!fix @Path("user") so we keep the possibilities to add registration of companies, accounts and whatever is needed